### PR TITLE
implementing initial vaccine data fixtures.

### PR DESCRIPTION
**Pull request by tiemfah**
So that we don't have to type in vaccine model every time.
type this after we run migration.
```bash
python manage.py loaddata vaccine/fixtures/vaccine.json
```

### CODE REVIEW

**Comment by tiemfah** [comment]
need a test case to test that loaddata works.

**Review by chiiida — CHANGES_REQUESTED** [review]
What about DoseModel? Can you also add JSON for DoseModel?
We also need DoseModel in order to test the whole model of vaccine and user.

**Comment by tiemfah** [comment]
Will merge into master directly after @llleyelll  merge her `vaccine-model-upgrade` branch.

**Pull request closed without merging**
